experimental/datacommons-js/README.md

```diff
@@ -0,0 +1,366 @@
+      statVar: "Count_Person_BelowPovertyLevelInThePast12Months",
+    },
+  ],
+});
+```
+
+Renders:
+
+![Bar chart](./assets/bar-chart.png "Bar Chart")
+
+### Line chart
+
+Draw a multi-variable line chart
+
+Initialize empty div
+
+```html
+<div id="line-example"></div>
+```
+
+Render the chart
+
+```js
+datacommons.drawLine(document.getElementById("line-example"), {
+  id: "line-chart-1",
+  svgChartHeight: 200,
+  className: "my-chart",
+  apiRoot: datacommons.root,
+  isDataTile: false,
+  title:
+    "Population Below Poverty Level Status in Past Year in United States (2020)",
+  place: {
+    dcid: "country/USA",
+    name: "United States of America",
+    types: ["Country"],
+  },
+  statVarSpec: [
+    {
+      name: "Population Below Poverty Level Status in Past Year",
+      statVar: "Count_Person_BelowPovertyLevelInThePast12Months",
+    },
+  ],
+});
+```
+
+Renders:
+![Line chart](./assets/line-chart.png "Line Chart")
+
+### Map chart
+
+Draw a choropleth map
+
+Initialize empty div
+
+```html
+<div id="map-example"></div>
+```
+
+Render the chart
+
+```js
+datacommons.drawMap(document.getElementById("map-example"), {
+  id: "map-chart-1",
+  svgChartHeight: 200,
+  className: "my-chart",
+  apiRoot: datacommons.root,
+  isDataTile: false,
+  title:
+    "Population Below Poverty Level Status in Past Year in States of United States (2020)",
+  place: {
+    dcid: "country/USA",
+    name: "United States of America",
+    types: ["Country"],
+  },
+  enclosedPlaceType: "State",
+  statVarSpec: {
+    name: "Population Below Poverty Level Status in Past Year",
+    statVar: "Count_Person_BelowPovertyLevelInThePast12Months",
+  },
+});
+```
+
+Renders:
+
+![Map chart](./assets/map-chart.png "Map Chart")
+
+### Ranking chart
+
+Draw ranking chart (bar chart + ordered table)
+
+Initialize empty div
+
+```html
+<div id="ranking-example"></div>
+```
+
+Render the chart
+
+```js
+datacommons.drawRanking(document.getElementById("ranking-example"), {
+  placeName: "USA",
+  placeType: "State",
+  withinPlace: "country/USA",
+  statVar: "Count_Person_BelowPovertyLevelInThePast12Months",
+  isPerCapita: false,
+  scaling: 1,
+  unit: "",
+  date: "2020",
+});
+```
+
+Renders:
+
+![Ranking chart](./assets/ranking-chart.png "Ranking Chart")
+
+## API
+
+### `datacommons.drawBar`
+
+Draws bar chart tile with specified [statistical variable](https://datacommons.org/tools/statvar).
+
+- `element` (HTMLElement)
+
+  DOM element to attach the chart
+
+- `props` (BarTilePropType)
+
+  Chart configuration object
+
+  - `id` (string)
+
+    DOM id attribute to attach to the chart
+
+  - `title` (string)
+
+    Chart title
+
+  - `place` (NamedTypedPlace)
+
+    Scope the statistical variable query by this place configuration
+
+    - `dcid` (string): Place DCID
+    - `name` (string): Place name
+    - `types` (string[]): [Place types](https://datacommons.org/browser/Place)
+
+  - `enclosedPlaceType` (string)
+
+    Enclosing place type
```

Comment:
enclosed? Not sure if this is confusing, maybe expand the meaning like "The type of places that are enclosed in the main place"

Reply:
Updated to:

`Show statistical variable observations at this place type. Must be enclosed in the parent place type.`